guide/getting-started/why-bitcoin-is-unique.md

```diff
@@ -60,19 +51,37 @@ We have already hinted at some of the unique properties of Bitcoin, but let’s
    layout = "float-right-desktop"
 %}
 
-The economic definition of money typically involves three specific [functions](https://en.wikipedia.org/wiki/Money#Functions) (medium of exchange, unit of account, store of value) and five [properties](https://en.wikipedia.org/wiki/Money#Properties) (durability, portability, fungibility, scarcity, divisibility, recognizability). Although bitcoin is a new breed of money, it still satisfies all of these requirements. Instead of relying on physical properties (like gold and silver) or trust in central authorities (like traditional currencies), bitcoin relies on a network of thousands of computers to enforce its mathematically defined ruleset.
+Historically, in order to be considered money, it must be usable as each of the following:
 
+1. A medium of exchange
+2. A unit of account
+3. A store of value
+
+In addition, six fundamental properties define whether or not a currency can actually function as money. Those [properties](https://en.wikipedia.org/wiki/Money#Properties) are:
+
+1. Durability
+2. Portability
+3. Fungibility
+4. Scarcity
+5. Divisibility
+6. Recognizability
+
+Although bitcoin is purely digital, it meets every classical definition of what makes something money. Instead of relying on physical properties (like gold and silver) or central authorities (like government-issued fiat currencies), bitcoin relies on the world’s most powerful computer network to mathematically enforce the rules that make it the first truly digital form of cash.
 </div>
 
 ## You own it
 
-When we carry cash, we are entirely in control of its value. When we purchase something and give the cash away, it is gone. Online banking and payments rely on third parties that hold our money for us and allow us to access it through their websites and applications. Bitcoin manages to combine the benefits of cash as a bearer instrument with online banking convenience. When you control your bitcoin private keys, nobody in the world can take your bitcoin.
+When you take cash out of your bank account, it becomes your responsibility. If you lose it, it’s lost. If you transact with it, no middleman can block the transaction. Either way, you are its custodian—this is bitcoin in a nutshell. It’s digital, self-custodial cash.
+
+Fiat currencies in digital form are different. Online banks and payment processors heavily depend on third-party websites and applications. Are they vetted? Will they responsibly manage your money? Are they even solvent? Tough to say. You just have to trust your bank’s judgment.
```

Comment:
I'm personally not a fan of throwing shade at banks. The banking system still works great for many purposes, and regulation is an important factor in stabilizing forces in a society. Also, 99% of people also just have to trust Bitcoin and Bitcoin application developers because they can't verify that the code is legit. My inclination would be to avoid attacking the character of banks and focus on how banking is not suited well for a variety of use cases (typically related to risk calculation and organizational overhead) and how Bitcoin simply works better for those.

Reply:
I don't think this section in particular throws shadow at banks? I think it actually compares stuff quite well?

Reply:
It's based on unfounded suspicion that banks are not trustworthy. It's an ad hominem attack - suggesting poor character rather than actually analyzing the trustworthiness of the two systems. It would be different if there were facts about vetting, solvency and a history of not being trustworthy by banks. You could easily write the same paragraph about Bitcoin ("Relies on open-source software. Is it vetted? Will the devs change the code and break Bitcoin? Tough to say...").

When using Bitcoin, you also have to relay on third-party websites and applications that non-technical people have no chance of vetting. So the same argument about whether banks are trustworthy applies to Bitcoin as well (it's just a different type of trust). I'd just leave the argument about trust out, as the section is about the ownership aspect (which is not about trusting the system, but you holding something compared to someone else holding it for you).

That got a bit long, but hope it makes sense. I just think it's a bit hand-wavy.

Reply:
@GBKS @sqcken  so any concrete suggestions here?  This one is a blocker to get this merged.

Reply:
My recommendation is to just remove that middle paragraph. The text works without it.

Reply:
@GBKS The first paragraph talks about fiat in the physical world and the second one complements it by giving the state of things in the digital world, but sure we can remove it, though I still think it gives nice overview on the state of things.

Reply:
```suggestion
```
OK, I'll remove it.